src/BuildIntegration/Microsoft.NETCore.Native.Unix.props

```diff
@@ -31,6 +32,7 @@ See the LICENSE file in the project root for more information.
   </ItemGroup>
 
   <ItemGroup>
+    <NativeLibrary Condition="'$(IlcMultiModule)' == 'true' and $(NativeCodeGen) == ''" Include="$(FrameworkLibPath)\libframework.a" />
```

Comment:
Oh, nevermind. I just noticed we only have that in `BuildFrameworkNativeObjects.proj`. We could pull that out into a separate .props file to maximize sharing, but that might be overkill.